api/src/main/kotlin/edu/wgu/osmt/richskill/RichSkillDocCsvExport.kt

```diff
@@ -0,0 +1,45 @@
+package edu.wgu.osmt.richskill
+
+import edu.wgu.osmt.config.AppConfig
+import edu.wgu.osmt.csv.CsvColumn
+import edu.wgu.osmt.csv.CsvResource
+import edu.wgu.osmt.jobcode.JobCode
+import edu.wgu.osmt.jobcode.JobCodeBreakout
+
+class RichSkillDocCsvExport(
```

Comment:
It looks like we are no longer using this class. If not, would you please remove it?

Reply:
removing..